src/QMCTools/QMCFiniteSize/SkParserHDF5.h

```diff
@@ -0,0 +1,24 @@
+#ifndef SK_PARSER_HDF5_H
+#define SK_PARSER_HDF5_H
+
+#include "QMCTools/QMCFiniteSize/SkParserBase.h"
+#include "Configuration.h"
+#include "io/hdf_archive.h"
+#include <vector>
+
+namespace qmcplusplus
+{
+class SkParserHDF5 : public SkParserBase
+{
+public:
+  SkParserHDF5();
+  ~SkParserHDF5();
```

Comment:
Please document this class.
No need of empty constructor and destructor.

Reply:
removed, and I documented the various other classes used in the qmcfinitesize tool